bazel/integration/index.bzl

```diff
@@ -6,10 +6,30 @@ def _serialize_file(file):
 
     return struct(path = file.path, shortPath = file.short_path)
 
+def _serialize_and_expand_location(ctx, value):
+    """Expands Bazel make location expressions for the given value. Returns a JSON
+      serializable dictionary matching the `BazelExpandedValue` type in the test runner."""
+    new_value = ctx.expand_location(value, targets = ctx.attr.data)
+
+    return {
+        "value": new_value,
+        "containsExpandedValue": new_value != value,
```

Comment:
If I understand correctly, while this is JSON serializable, its not actual put into JSON, so does this properly match the typings expect?

I would think that this is:
```ts
{
  value: string;
  containsExpandedValue: 'True' | 'False';
}
```

rather than 
```ts
{
  value: string;
  containsExpandedValue: boolean;
}
```

If this is the case and '"False"' is provided, if we expect a boolean, then we are going to get alot of unexpected results.

Reply:
That is what I meant, I just was unsure of where the dictionary gets encoded as JSON in the process and wanted to make sure to confirm